core/src/main/java/org/frankframework/senders/ReconnectSenderWrapper.java

```diff
@@ -65,20 +65,36 @@ public void open() throws SenderException {
 			sender.open();
 			super.open();
 		} finally {
+			super.close();
```

Comment:
It should only close the sender, not it self.

Reply:
Yes, you're right. I'll remove it.